pandas/tests/plotting/test_misc.py

```diff
@@ -77,6 +77,12 @@ def scat(**kwds):
             _check_plot_works(scat, diagonal='hist')
         with tm.assert_produces_warning(UserWarning):
             _check_plot_works(scat, range_padding=.1)
+        with tm.assert_produces_warning(UserWarning):
+            _check_plot_works(scat, color='rgb')
```

Comment:
Purely for aesthetic purposes. I have trouble seeing some colors and wanted to visually confirm these changes rendered the same objects.